docs/certificatesigningrequest-metrics.md

```diff
@@ -2,6 +2,7 @@
 
 | Metric name| Metric type | Labels/tags | Status |
 | ---------- | ----------- | ----------- | ----------- |
+| kube_certificatesigningrequest_annotations | Gauge | `certificatesigningrequest`=&lt;certificatesigningrequest-name&gt; <br> `signer_name`=&lt;certificatesigningrequest-signer-name&gt;| STABLE |
```

Comment:
Considering that this is a new metric, we should make it `EXPERIMENTAL` for the time being. This comment also applies to the documentations.

Reply:
Done